### PR TITLE
Fix tab content hidden

### PR DESCRIPTION
**Pull request by Shivam-Samant**

### CODE REVIEW

File: resources/views/hr/application/put-on-hold-modal.blade.php [hunk 1]
```diff
@@ -17,18 +17,18 @@
...
-                <input type="checkbox" name="send_mail_to_applicant[hold]" class="custom-control-input show" id="holdSendMailToApplicant">
+                <input type="checkbox" name="send_mail_to_applicant[hold]" class="custom-control-input" id="holdSendMailToApplicant">
                 <label class="custom-control-label" for="holdSendMailToApplicant" >Send email</label>
                 </div>
             </div>
             </div>
             <div class="container">
                 <div>
-                    <ul class="nav nav-tabs menu">
+                    <ul class="nav nav-tabs" id="menu">
```
Comment (gautam071) [inline resources/views/hr/application/put-on-hold-modal.blade.php:27]: @Shivam-Samant Can you change the ID name relatable

**Review by gautam071 — CHANGES_REQUESTED** [review]
Looks good!

@P4NK4J Can you take a look at it once?

**Review by P4NK4J — APPROVED** [review]
Looks good,

**Review by gautam071 — APPROVED** [review]
Looks good

**Pull request merged**